components/NotificationBar.js

```diff
@@ -117,6 +150,20 @@ class NotificationBar extends React.Component {
           <div className={`NotificationLine ${status}`}>
             <h1>{title}</h1>
             <p className="description">{description}</p>
+
+            {isHostAdmin && (
+              <AcceptRejectBanner>
+                <AcceptRejectButtons collective={collective} host={host} />
+                {this.state.showRejectionModal && (
+                  <AppRejectionReasonModal
+                    show={this.state.showRejectionModal}
+                    onClose={() => this.setState({ showRejectionModal: false })}
+                    collectiveId={this.state.collectiveId}
```

Comment:
I don't see where `state.collectiveId` is set in this component, my understanding is that its value will always be `null`.

Reply:
I have removed it since it is not needed here